CHANGELOG.md

```diff
@@ -8,6 +8,103 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Eclipse Data Space Connector (Tractus-X EDC)](https://github.com/eclipse-tractusx/tractusx-edc) | [0.7.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.3) | [0.7.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.3) |
+| [EcoPass Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/eco-pass-kit) | n/a | [1.5.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/Eco_Pass_KIT/changelog) |
+| [ESS Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/ess-kit) | n/a | [0.3.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/ESS-Kit/ESS%20Kit%20Changelog) |
+| [Golden Record Business Partner Number (BPN) Service](https://github.com/eclipse-tractusx/bpdm) | bpdm: [5.0.3](https://github.com/eclipse-tractusx/bpdm/releases/tag/bpdm-5.0.3) | [6.1.0](https://github.com/eclipse-tractusx/bpdm/releases/tag/v6.1.0) |
+| [Identity and Access Management (IAM)](https://github.com/eclipse-tractusx/portal-iam) | centralidp: [3.0.1](https://github.com/eclipse-tractusx/portal-iam/releases/tag/centralidp-3.0.1)<br>sharedidp: [3.0.1](https://github.com/eclipse-tractusx/portal-iam/releases/tag/sharedidp-3.0.1) | keycloak: [23.0.7](https://www.keycloak.org/docs/latest/release_notes/index.html#keycloak-23-0-7) |
+| [Industry Core KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/industry-core-kit) | n/a | [1.1.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/Industry%20Core%20Kit/Industry%20Core%20Kit%20Changelog) |
+| [Item Relationship Service (IRS)](https://github.com/eclipse-tractusx/item-relationship-service) | item-relationship-service: [7.4.0](https://github.com/eclipse-tractusx/item-relationship-service/releases/tag/item-relationship-service-7.4.0) | [5.4.0](https://github.com/eclipse-tractusx/item-relationship-service/releases/tag/5.4.0) |
+| [Knowledge Agents](https://github.com/eclipse-tractusx/knowledge-agents) | remoting-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/remoting-agent-1.13.22)<br>provisioning-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/provisioning-agent-1.13.22)<br>conforming-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/conforming-agent-1.13.22)<br>matchmaking-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/matchmaking-agent-1.13.22) | remoting-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.13.22)<br>provisioning-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.13.22)<br>conforming-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.13.22)<br>matchmaking-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.13.22) |
```

Comment:
It is a very good idea to aggregate components like that. We could even simplify a bit more

| [Knowledge Agents](https://github.com/eclipse-tractusx/knowledge-agents) | remoting-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/remoting-agent-1.13.22)<br>provisioning-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/provisioning-agent-1.13.22)<br>conforming-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/conforming-agent-1.13.22)<br>matchmaking-agent: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/matchmaking-agent-1.13.22) | [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.13.22) |

| [Knowledge Agents EDC](https://github.com/eclipse-tractusx/knowledge-agents-edc) | agent-plane: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents-edc/releases/tag/agent-plane-1.13.22)<br>agent-plane-azure-vault: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents-edc/releases/tag/agent-plane-azure-vault-1.13.22) | [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents-edc/releases/tag/v1.13.22) |

Reply:
@drcgjung THX, adapted the PR and also the Table in the description. Please have a look. Should be fine right now. Thumbs up?